eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -101,6 +101,9 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('security.yml');
         // Slots
         $loader->load('slot.yml');
+        if (interface_exists('FOS\JsRoutingBundle\Extractor\ExposedRoutesExtractorInterface')) {
```

Comment:
I had no issues with that approach in TagsBundle.

AFAIK the list of bundles is initialized before the extensions are ran.